controllers/admin/AdminProductsController.php

```diff
@@ -39,7 +39,6 @@ class AdminProductsControllerCore extends AdminController
      *          As of 1.5 it is recommended to not set a limit to max image size
      */
     protected $max_file_size = null;
-    protected $max_image_size = null;
```

Comment:
Shouldn't this be moved into the parent `AdminController` no problem with using an undefined dynamic property in PHP 8.2?

Reply:
Check AdminController

Reply:
![image](https://github.com/PrestaShop/PrestaShop/assets/1533248/8ea8a74a-4679-4799-85d1-e9cc5e457f02)